plugin/core/settings.py

```diff
@@ -112,7 +112,10 @@ def remove_for_testing(self, config: ClientConfig) -> None:
         if self._listener:
             self._listener()
 
-    def add_external_config(self, name: str, s: sublime.Settings) -> None:
+    def add_external_config(self, name: str, s: sublime.Settings, file: str) -> None:
+        base = sublime.decode_value(sublime.load_resource(file))
```

Comment:
It should be saved somewhere, but because add_external_config and remove_external_config are actually useless (because of #899), I'll add that later.